README.md

```diff
@@ -113,6 +113,16 @@ Perhaps Goby should be far easier for Rubyists to comprehend. You can use Ruby's
 - Flow control
     - `if`, `else`, `elsif`
     - `while`
+- Literals
+    - Numeric literal
+        - Integer: `100`, `(-299)`, `(-0)`
```

Comment:
@hachi8833  I just found that in current master we don't need to wrap parentheses around negative integer or float anymore (at least in `igb`). Can you help me confirm this? (when did we fix this?)

Reply:
I confirmed that bare negative integer/float works as you pointed out, except `to_class`.
This will be fixed #557.